cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -881,15 +934,16 @@ class ClassTemplateInstantiation extends Class {
   /**
    * Gets the class template from which this instantiation was instantiated.
    *
-   * Example: For `std::vector<float>`, returns `std::vector<T>`.
+   * For example: For `std::vector<float>`, the result is `std::vector<T>`.
    */
   TemplateClass getTemplate() {
     result = tc
   }
 }
 
 /**
- * A specialization of a class template.
+ * A specialization of a class template (this may be a full or partial template
+ * specialization).
```

Comment:
I think this ought to have examples of both, or perhaps links to the Full and Partial subclasses.

Reply:
Links added.